### DISCUSSION TITLE
Failure of non-linux draft tests

### DISCUSSION THREAD

**Discussion by purva-thakre** [unitaryfund/mitiq#1973] [General]
I have a WIP PR. I know it is failing 'validate' because I have ignored to account for type checking. 

Even then, all of the non-linux Windows tests are failing with a similar message across all of them. Is this unrelated to the PR and should be ignored for now?

https://github.com/unitaryfund/mitiq/actions/runs/5959506725/job/16165346786?pr=1963
https://github.com/unitaryfund/mitiq/actions/runs/5959506725/job/16165346846?pr=1963
https://github.com/unitaryfund/mitiq/actions/runs/5959506725/job/16165346917?pr=1963
https://github.com/unitaryfund/mitiq/actions/runs/5959506725/job/16165347000?pr=1963

Edit: Tried a bunch of different `fetch depth` options. All of them led to the Windows draft build failing. Why does the build for mac pass but fail for windows?

Edit2: If `fetch_depth = 2`, mac build passes but windows build fails. If `fetch_depth = 0`, windows build passes while mac build fails. I think this is unrelated to my current PR. 

I followed this [answer on StackOverflow](https://stackoverflow.com/a/76769122/10241324) to set `fetch_depth = 0`.

**Comment by natestemen** [comment]
It looks like these failures are related to the "draft" workflow that @Aaron-Robertson added. I would personally convert the PR out of draft mode to get "regular" CI running.

This might be something Aaron wants to look into.

**Reply by purva-thakre** [reply]
IDK. The issue fixed itself this afternoon. https://github.com/unitaryfund/mitiq/pull/1975 

~~As an aside, how do I close a discussion? I have the option to lock it which is what I did once I knew the issue was unrelated to what I am working on.~~ never mind. I just saw the option for it.

**Comment by Misty-W** [comment]
Issue seems to have fixed itself.
